src/App.js

```diff
@@ -9,32 +9,25 @@ import { fieldSearch } from "./search";
 
 import "./App.css";
 
-const mentorIds = mentors.map((mentor, index) => index);
-
-// Convert names into usable `#`-ids.
-const mentorHashes = mentors.map(({ name }) =>
-  name.toLowerCase().replace(" ", "-").replace("(", "").replace(")", "")
-);
 const setHash = (hash) => window.history.replaceState({}, "", `#${hash}`);
 
 function App() {
-  const [visibleMentorIds, setVisibleMentorIds] = useState(mentorIds);
+  const [visibleMentorIds, setVisibleMentorIds] = useState(
+    Object.keys(mentors)
+  );
   const searchChangeHandler = (input) =>
-    input.trim().length === 0 && setVisibleMentorIds(mentorIds);
+    input.trim().length === 0 && setVisibleMentorIds(Object.keys(mentors));
```

Comment:
This should probably be stored in a variable since it’s going to be used multiple times.

Reply:
Hmm, I disagree on this part if we were to store this in a global constant, as I think it would be confusing to have an array that is essentially a "subset" of this object. But that said, if it's used multiple times in a single file then I agree it should be made a constant. Iirc it's only used on this line, but that may no longer be the case in your code.

Reply:
It’s used at the start of the function, and I thought maybe running `Object.keys` every time the search is cleared might not be efficient.

Reply:
Ohh yeah makes sense. Will amend that.